### PR TITLE
Improve fatal errors; improve display of messages around commands

### PR DESCRIPTION
**Pull request by DavidS-ovm**
With error:
2024-05-23_15MS+0200_875x173

Without error, before command:
2024-05-23_15MS+0200_836x187

Without error, after command:
2024-05-23_15MS+0200_683x257

### CODE REVIEW

File: cmd/tea_taskmodel.go [hunk 1]
```diff
@@ -92,6 +92,12 @@ func (m taskModel) Update(msg tea.Msg) (taskModel, tea.Cmd) {
 		if m.spinner.ID() == msg.id {
 			m.status = msg.status
 		}
+	case fatalError:
+		if m.spinner.ID() == msg.id {
+			m.status = taskStatusError
+			m.title = fmt.Sprintf("%v: %v", m.title, msg.err)
```
Comment (DavidS-ovm) [inline cmd/tea_taskmodel.go:98]: It made sense when I wrote it to replicate (the start of) the error message here. Looking at the screenshot I have doubts. Any opinion on it, @dylanratcliffe ?

Comment (dylanratcliffe) [reply cmd/tea_taskmodel.go:98]: I don't live the double-pipe indentation thing TBH. A single one is okay as it mirrors terraform. Also could do with one or two less lines of whitespace I think before and after

Comment (DavidS-ovm) [reply cmd/tea_taskmodel.go:98]: done.

**Review by dylanratcliffe — APPROVED** [review]
Much nicer

**Pull request merged**